packages/next-app/pages/api/auth/[...nextauth].ts

```diff
@@ -13,6 +21,41 @@ export default NextAuth({
 				url: "https://api.twitter.com/2/users/me",
 				params: { "user.fields": ["created_at", "public_metrics"] },
 			},
+			profile({ data }) {
+				store.set("auth-provider", "twitter");
```

Comment:
where does this `store` store the data when it's on API side?

Reply:
`store` picks the best of a few available types and falls back to whichever works, so I'm guessing one of session/userData/global storage?
<img width="747" alt="Screen Shot 2022-03-15 at 6 47 02 PM" src="https://user-images.githubusercontent.com/52016903/158315085-ddbacaa8-2ee2-4cc7-9c7c-39a80b18a63e.png">

Reply:
as discussed there might be issue with `store` in serverless environment. Need to use Redis for it, this change can be on the next PR however